### PR TITLE
Updated IDL Types in HVAC_Telemetry.xml (sorry that I didn't do that …

### PR DESCRIPTION
**Pull request by wvreeven**
…before). Added the HVAC_Commands.xml file.

### CODE REVIEW

File: sal_interfaces/HVAC/HVAC_Commands.xml [hunk 1]
```diff
@@ -1,30 +1,309 @@
...
-        <EFDB_Name>minValveFlow</EFDB_Name>
-        <Description>Minimum flow.</Description>
-        <IDL_Type>double</IDL_Type>
-        <Units>m3 / s</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand></SALCommandSet>
\ No newline at end of file
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl'                                              href='http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl'?>
```
Comment (r-owen) [inline sal_interfaces/HVAC/HVAC_Commands.xml]: Why this change? The added spaces before `href` is surprising.

File: sal_interfaces/HVAC/HVAC_Commands.xml [hunk 2]
```diff
@@ -1,30 +1,309 @@
...
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_command_lsst_fwdsl_piso01_fwdsl_tcc_dash_p1_dash_sala_dash_maquinas</EFDB_Topic>
```
Comment (r-owen) [inline sal_interfaces/HVAC/HVAC_Commands.xml]: Unfortunately we still need the `Alias` tags in Commands and Events (but not Telemetry) files with SAL 4.0. We can delete them when we switch to SAL 4.1.

You should see an error if you run `make_idl_files.py HVAC` with SAL 4.0

File: sal_interfaces/HVAC/HVAC_Commands.xml (hunk 1 above)
Comment (wvreeven) [reply sal_interfaces/HVAC/HVAC_Commands.xml]: This was due to a coding error of mine where I wrapped a long string over two lines and indented it. That has been fixed now.

File: sal_interfaces/HVAC/HVAC_Commands.xml (hunk 2 above)
Comment (wvreeven) [reply sal_interfaces/HVAC/HVAC_Commands.xml]: I added the Alias tag to all Commands.

**Comment by wvreeven** [comment]
I committed new files with the issues fix. Can you please verify that all is well now?

File: sal_interfaces/HVAC/HVAC_Commands.xml [hunk 3]
```diff
@@ -1,30 +1,317 @@
...
\ No newline at end of file
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type='text/xsl' href='http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl'?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_command_lsstBarraoblPiso01BarraoblTccGuionP1GuionSalaGuionMaquinas</EFDB_Topic>
+    <Alias>lsstBarraoblPiso01BarraoblTccGuionP1GuionSalaGuionMaquinas</Alias>
+    <item>
+      <EFDB_Name>lsstBarraoblPiso01BarraoblVea_01BarraoblComandoEncendidoLsst</EFDB_Name>
```
Comment (r-owen) [inline sal_interfaces/HVAC/HVAC_Commands.xml]: Is there an easy way to get rid of these remaining underscores?

Comment (wvreeven) [reply sal_interfaces/HVAC/HVAC_Commands.xml]: Possibly yes. I'll open an issue for it because I will need to think about how to do that without introducing too much "noise".
